### PR TITLE
Report Correct Line Numbers for esnext

### LINKED ISSUE #12: Coverage is against transpiled ES5 Code
When using ES6 in my project (also using ES7 decorators via `ember-computed-decorators`), the code coverage being reported is against the transpiled ES5 code. In order to make the coverage report more useful when using services such as *Coveralls* or editor plugins such as *lcov-info* for Atom, the coverage should report against line numbers in the original source code.

### PR DESCRIPTION
**Pull request by sandersky**
This reverts commit d51c6702c208334046ad7a712cabb173f26c309a and resolves #12.

This change uses the applications babel configuration for the instrumenter's configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Configuration is optional. It should be put in a file at `config/coverage.js`.
 
 - `coverageFolder`: Defaults to `coverage`. A folder relative to the root of your project to store coverage results.
 
+- `useBabelInstrumenter`: Defaults to `false`. Whether or not to use Babel instrumenter instead of default instrumenter. The Babel instrumenter is usefull when you are using features of ESNext as it uses your Babel configuration defined in `ember-cli-build.js`.
```
Comment (rwjblue) [inline README.md]: Small typo: s/usefull/useful/

Comment (sandersky) [inline README.md]: Nice catch!

**Comment by rwjblue** [comment]
Thanks @sandersky!

**Comment by rwjblue** [comment]
Released as v0.2.1

**Pull request merged**